packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -58,6 +58,34 @@ class Evaluation {
 
 /// An accessibility guideline describes a recommendation an application should
 /// meet to be considered accessible.
+///
+/// Use [meetsGuideline] matcher to test whether a screen meets the
+/// accessibility guideline.
+///
+/// {@tool snippet}
+///
+/// This sample demonstrates how to run an accessibility guideline in a unit
+/// test against a single screen.
+///
+/// ```dart
+/// testWidgets('HomePage meets androidTapTargetGuideline', (WidgetTester tester) async {
+///   final SemanticsHandle handle = tester.ensureSemantics();
+///   await tester.pumpWidget(const MaterialApp(home: HomePage()));
+///   await expectLater(tester, meetsGuideline(androidTapTargetGuideline));
+///   handle.dispose();
+/// });
+/// ```
+/// {@end-tool}
+///
+/// See also:
+///  * [androidTapTargetGuideline]: which provides android specific guidance
```

Comment:
maybe more specific: `which checks that tappable nodes have a minimum size of 48 by 48 pixels.`